### PR TITLE
Bulk edit improvements

### PR DESCRIPTION
**Pull request by szok**
Added:
- more fields to Back Office asset
- exclude fill fields in bulk edit

Connected to #1690

### CODE REVIEW

File: src/ralph/data_center/admin.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ class DataCenterAssetAdmin(
     multiadd_info_fields = list_display + ['rack']
     one_of_mulitvalue_required = ['sn', 'barcode']
     bulk_edit_list = list_display
+    bulk_edit_list_exclude = ['barcode', 'sn']
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: so exclude or no_fillable?

File: src/ralph/admin/mixins.py [hunk 2]
```diff
@@ -203,6 +203,19 @@ def formfield_for_manytomany(self, db_field, request=None, **kwargs):
             return db_field.formfield(**kwargs)
         return super().formfield_for_manytomany(db_field, request, **kwargs)
 
+    def get_changelist_form(self, request, **kwargs):
```
Comment (mkurek) [inline src/ralph/admin/mixins.py:208]: could you add some docstring here (what this method do, why this weird class `vTextField no-fillable` etc)?

Comment (szok) [inline src/ralph/admin/mixins.py:208]: Done :)

Comment (mkurek) [inline src/ralph/admin/mixins.py:208]: i mean description of bulk_edit_no_fillable option (you could btw add empty list as default value in this class and describe it there)

Comment (szok) [inline src/ralph/admin/mixins.py:208]: OK, done

**Comment by ar4s** [comment]
:ok:

**Comment by vi4m** [comment]
Nicely done!

**Comment by vi4m** [comment]
LGTM

**Pull request merged**